test/monitoring/scrape_targets_test.go

```diff
@@ -37,6 +37,10 @@ func testScrapeTargetRechability(t *testing.T, v1api v1.API) {
 	); err != nil {
 		t.Errorf("%v", err)
 
+		if w == nil {
```

Comment:
if you use `w = tabwriter.NewWriter(os.Stdout, 16, 8, 2, '\t', 0)` in line 33, and 63, then we don't need to do this logic. And you save one line in 63. :smile:

Reply:
But we don't want to print anything if the test fails. Hence returning from here.

Reply:
Right, it's not obvious from the code, but it make sense :+1: